src/main.rs

```diff
@@ -115,11 +115,20 @@ fn load_script(script: &str) -> io::Result<(String, String)> {
 
     log::trace!("Reading script from {}", script);
 
-    match fs::read_to_string(script) {
+    let file_path = Path::new(script);
+    let file_contents = match file_path {
+        path if path.is_file() => fs::read_to_string(path),
```

Comment:
That makes sense, I wasn't aware of this type of bug before so I'll make sure to add that to my memory! Thanks!
I was initially trying to match for `io::ErrorKind::IsADirectory` but it seemed to not be stabilized yet so that's why I ended up with the initial solution. Checking for `is_dir` after looks much nicer 👍